### PR TITLE
docs: Add backing image related scalability test

### PR DESCRIPTION
**Pull request by shuo-wu**
Longhorn/longhorn#3242

ref: https://github.com/longhorn/longhorn/issues/3242#issuecomment-984409310

### CODE REVIEW

**Comment by PhanLe1010** [comment]
One quick question, why do you create 10 backing images but only use 5 in the SCs? What is the intention?

**Comment by shuo-wu** [comment]
> One quick question, why do you create 10 backing images but only use 5 in the SCs? What is the intention?

Before the fix, the extra backing images will lead to replica controller enqueuing regardless of it is not used. Hence I want to test if the unused backing images still affect the enqueuing.

File: docs/content/manual/release-specific/v1.2.3/test-scalability-with-backing-image.md [hunk 1]
```diff
@@ -0,0 +1,338 @@
...
+            requests:
+              storage: 32Mi
+    ```
+    
+   Notice:  
+     - Each replica/pod of this StatefulSet contains 10 volumes. You just adjust the volume count by scaling this StatefulSet.
+     - Launching this StatefulSet will take a long time.
+5. Wait for all pods up and running.
+6. Create and attach a volume via UI. This should not take a long time.
+7. Upgrade the longhorn manager version to `v1.2.3-rc1`. Notice that each longhorn manager pod will take some time to walk through the ugprade path. Please do not operate Longhorn before each manager pod printing out the log like `"Debug Server listening on 127.0.0.1:6060" node=shuo-k8s-worker1`.
```
Comment (innobead) [inline docs/content/manual/release-specific/v1.2.3/test-scalability-with-backing-image.md:336]: For step7 to 8, I think we don't need to include in this test case, but just have a note this test case will fail in the previous version (< 1.2.3 or 1.3.0 if the issue does not be fixed in 1.2.3)

**Review by innobead — CHANGES_REQUESTED** [review]
LGTM, but few comments needed to resolve and also what Phan asked is suggested to explain in this test case well to give more context for QA.

**Review by PhanLe1010 — APPROVED** [review]

**Review by PhanLe1010 — DISMISSED** [review]
LGTM

File: docs/content/manual/release-specific/v1.2.3/test-scalability-with-backing-image.md (hunk 1 above)
Comment (shuo-wu) [reply docs/content/manual/release-specific/v1.2.3/test-scalability-with-backing-image.md:336]: Got it. I will remove these 2 steps then add more comments for explanation.

**Review by innobead — APPROVED** [review]

**Comment by kaxing** [comment]
@shuo-wu, thanks for the clear steps and context!
One idea, can we add those manifests text block into actual yaml file? It will also make deploying the workload easier.

**Comment by shuo-wu** [comment]
> @shuo-wu, thanks for the clear steps and context! One idea, can we add those manifests text block into actual yaml file? It will also make deploying the workload easier.

I am not sure if it would affect the directory structure. If NO, YES we can.

**Pull request merged**